### PR TITLE
refactor(routes): remove preVerifyToken support

### LINKED ISSUE #1599: Remove preVerifyToken support
We haven't used it in a long time and have no reason to believe we'll use it again, see https://github.com/mozilla/fxa-auth-server/issues/796#issuecomment-268433225 and related discussion.  Let's remove it to simplify the code.

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #1599 

DO NOT MERGE yet...

### CODE REVIEW

**Comment by rfk** [comment]
FWIW this looks good to me, it's not obvious how that change here can break those tests, AFAICT they're not using any of the pre-veification stuff.

**Comment by vladikoff** [comment]
> FWIW this looks good to me, it's not obvious how that change here can break those tests, AFAICT they're not using any of the pre-veification stuff.

hah yeah that's what i was thinking, except all signs point that something broke them :P

**Comment by rfk** [comment]
Heh, yeah...just wish I had a suggestion for how to debug :-(

File: test/remote/account_preverified_token_tests.js [hunk 1]
```diff
@@ -1,131 +0,0 @@
...
-function nowSeconds() {
-  return Math.floor(Date.now() / 1000)
-}
-
-describe('remote account preverified token', function() {
-  this.timeout(15000)
-  let server
-  before(() => {
-    process.env.TRUSTED_JKUS = 'http://127.0.0.1:9000/.well-known/public-keys'
-    process.env.IP_PROFILING_ENABLED = true
```
Comment (rfk) [inline test/remote/account_preverified_token_tests.js]: My theory is, this line used to execute before the failing tests, and it meant all subsequent tests could skip sign-in confirmation.  With it removed, the failing tests are now trying to access keys without verifying their signin, producing the "unverified account" errors.

**Comment by rfk** [comment]
@vladikoff I updated the tests to cope with sign-in confirmation, and rebased-in some more fixes from master; let's see if this passes...

**Comment by rfk** [comment]
Ugh, latest failures appear to be due to errors in the db-server...

**Comment by rfk** [comment]
@vladikoff this works for me now, when run atop the fix from https://github.com/mozilla/fxa-auth-db-mysql/pull/215; can you please take a look and merge if you're happy with it?

**Comment by vladikoff** [comment]
@rfk thanks for pushing this forward, will try to take a look !

**Pull request merged**